src/app/rpmostree-container-builtins.c

```diff
@@ -537,6 +537,7 @@ rpmostree_container_builtin_upgrade (int argc, char **argv, GCancellable *cancel
 
     if (!rpmostree_context_assemble_commit (rocctx->ctx, tmprootfs_dfd, NULL,
                                             NULL, &new_commit_checksum,
+                                            TRUE,
```

Comment:
This should be the fifth arg, not sixth.

Also, not directly related to just this, but for bool params, I like appending a `/* param-name */` to help readers know what the flag does. It quickly becomes confusing when multiple bool params are passed. But it's up to you.